src/js/components/Select/Select.js

```diff
@@ -271,8 +273,13 @@ const Select = forwardRef(
   },
 );
 
-Select.defaultProps = { ...defaultProps };
-
+Select.defaultProps = {
+  closeOnChange: true,
+  dropAlign: { top: 'bottom', left: 'left' },
+  messages: { multiple: 'multiple' },
```

Comment:
What is this message? when would I see it?

Reply:
sorry this must be a bad merge. are the other ones obselete as well?

Reply:
not sure what you mean.